packages/flutter_tools/test/src/context.dart

```diff
@@ -60,7 +60,8 @@ void _defaultInitializeContext(AppContext testContext) {
     ..putIfAbsent(SimControl, () => new MockSimControl())
     ..putIfAbsent(Usage, () => new MockUsage())
     ..putIfAbsent(FlutterVersion, () => new MockFlutterVersion())
-    ..putIfAbsent(Clock, () => const Clock());
+    ..putIfAbsent(Clock, () => const Clock())
```

Comment:
This is already added in `_defaultInitializeContext()`

Reply:
Not sure what you mean; this _is_ `_defaultInitializeContext()`.

Reply:
That's what I get for reviewing when I'm tired - I read this as a new line instead of a mutated line 😪